pds_doi_core/input/node_util.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/python
+#
+#  Copyright 2020, by the California Institute of Technology.  ALL RIGHTS
+#  RESERVED. United States Government Sponsorship acknowledged. Any commercial
+#  use must be negotiated with the Office of Technology Transfer at the
+#  California Institute of Technology.
+#
+#------------------------------                                                                                                 
+
+import os
+import sys
+
+from pds_doi_core.util.general_util import get_logger
+
+# Get the common logger and set the level for this file.
+import logging
+logger = get_logger('pds_doi_core.input.node_util')
+
+class NodeUtil:
+    # This class NodeUtil provide services to look up a short name for a long name of the node id.
+
+    m_node_id_dict = {}
+
+    def __init__(self):
+        self.m_node_id_dict = {'ATM' :'Atmospheres',
+                               'ENG' :'Engineering',
+                               'GEO' :'Geosciences',
+                               'IMG' :'Cartography and Imaging Sciences Discipline',
```

Comment:
Extremely minor, so no change necessary: I'm slightly curious as to how IMG is the only Node to have "Discipline" associated with its name?  We are a Discipline Node, to be sure, but aren't the others (excepting EN and NAIF) as well?
BTW: Thanks for using `IMG` as that shorthand is used pervasively.

Reply:
👍

Reply:
Thanks Myche for contributing to the review. The name came actually from the PDS4 dictionnary at https://pds.nasa.gov/datastandards/documents/dd/current/PDS4_PDS_DD_1D00.html#d5e13937
This is indeed inconsistent with the others. If we want to change that that would a request for change for the PDS4 information model (https://github.com/NASA-PDS/pds4-information-model).